### ISSUE TITLE
Powah - Lens of Ender is unobtainable

### ISSUE THREAD

**Issue by MuteTiefling** [NillerMedDild/Enigmatica5#369]
**Modpack version**
0.4.15

**Issue**
Dunno if I'm just doing this wrong, but the JEI information tab for this says to use a Photoelectric Pane on an Enderman or Endermite. In both cases, right clicking either of these mobs with a Photoelectric pane causes the item to be removed from my inventory, the mob disappears (death noises are made), and no lens is given. 

**Logs**
[latest.log](https://github.com/NillerMedDild/Enigmatica5/files/4810988/latest.log)

Right at the end of the log I cheated in a photoelectric pane and an enderman spawn egg, then used them to attempt to craft the lens. Doesn't appear to be generating any errors.

**Comment by NielsPilgaard** [comment]
Thank you for the report, I will investigate the issue.

**Comment by NielsPilgaard** [comment]
Hmm, works just fine for me, were you on a server?

**Comment by MuteTiefling** [comment]
Nope, single player world. Not open to lan.

**Comment by NielsPilgaard** [comment]
.. huh.. 
Can you still do it?

**Comment by MuteTiefling** [comment]
Yep: https://streamable.com/4j12c7

The enderman just goes poof. No sign of them. I've even tried doing it in the end on a platform in the void so I could see if it is just teleporting away. There's simply no sign of it afterwards. 

<blockquote><img src="https://cdn-b-east.streamable.com/image/4j12c7.jpg?token=LYp9m9Lha3I9hvUY1rt15g&expires=1593120960" width="48" align="right"><div>Streamable</div><div><strong><a href="https://streamable.com/4j12c7">Enigmatica 5 - Singleplayer 2020-06-25 15-33-10</a></strong></div><div>Watch "Enigmatica 5 - Singleplayer 2020-06-25 15-33-10" on Streamable.</div></blockquote>

**Comment by NielsPilgaard** [comment]
Your backpack isn't picking it up automatically or something right? Could you try without other stuff in your inventory, and without baubles/armor?

**Comment by MuteTiefling** [comment]
Well. That's the weirdest thing ever. 

I took off all of my armor/gear/emptied my inventory and it immediately worked. 

I then put things back one thing at a time, testing with a new egg and new photoelectric pane.... and it worked every single time.  Even after restarting. Not a clue what happened there, but it certainly wasn't going in my backpack or dank nulls

Edit: I should note I have since updated to 0.5.1 So maybe some weird issue was fixed already by the mod author.
